tests/installation/partitioning/msdos_partition_table.pm

```diff
@@ -22,10 +22,10 @@ sub run {
     my $partitioner = $testapi::distri->get_expert_partitioner;
     $partitioner->run_expert_partitioner;
     foreach my $disk (@{$test_data->{disks}}) {
-        $partitioner->create_new_partition_table({disk => $disk->{name},
-                table_type => $disk->{table_type}, accept_deleting_current_devices_warning => 1});
+        $partitioner->create_new_partition_table({name => $disk->{name},
+                table_type => $disk->{table_type}, accept_deleting_current_devices_warning => 0});
         foreach my $partition (@{$disk->{partitions}}) {
-            $partitioner->add_partition_msdos({disk => $disk->{name},
+            $partitioner->add_partition_msdos({name => $disk->{name},
```

Comment:
As I can see from the `sub add_partition_msdos`, the `disk` method attribute should be kept, as it is used in the function. Or, in case you decide to call it `name`, I guess it should be also changed [there](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/1a3a432b6f8778a3d8d61f613d85f0203a4d9e14/lib/Installation/Partitioner/LibstorageNG/v4/ExpertPartitionerController.pm#L202) to `name`.

Reply:
That sub needs to be rewritten anyway.